docs/kit/operation-view/page10_extensions.md

```diff
@@ -0,0 +1,49 @@
+# EDC Extensions
+
+The following extensions provide additional functionality to the core EDC.
+They are currently only available in TractusX EDC.
+
+### Business Partner Validation
+
+This extension allows for validation of business partners after the creation of a contract.
```

Comment:
After? This will be used in the access policies, so before the contract offer will be send to the consumer.

Reply:
I thought access policies were checked before data transfer and after a contract is formed?

Reply:
Nope